src/test/mir-opt/const_prop/ref_deref.rs

```diff
@@ -6,15 +6,17 @@ fn main() {
 // START rustc.main.ConstProp.before.mir
 // bb0: {
 //     ...
-//     _2 = &(promoted[0]: i32);
+//     _4 = const main::promoted[0];
```

Comment:
how is the printing working if the printing logic doesn't contain `::`?

Reply:
Because the printing happens here https://github.com/rust-lang/rust/pull/67000/files#diff-54476e827f95a262ee4ee5251e600f7bR897 you pointed out to a fix on librustdoc :).